classes/class-p4-register-metabox.php

```diff
@@ -0,0 +1,704 @@
+<?php
+/**
+ * P4 Register Metabox Class
+ *
+ * @package P4MT
+ */
+
+if ( ! class_exists( 'P4_Register_Metabox' ) ) {
```

Comment:
I think it is time to get rid of these checks since autoloading is handled by master_site class.
It adds extra indentation without any reason.

Also class names should be nouns to define better their existance. A better name would be P4_Metabox_Register or P4_Metabox_Registrator

Reply:
yes It adds extra indentation, removed the `class_exists()` check and renamed the class name to `P4_Metabox_Register`